gum/arch-arm64/gumarm64writer.c

```diff
@@ -872,6 +868,24 @@ gum_arm64_writer_put_load_store_pair_post (GumArm64Writer * self,
       (rt2 << 10) | (rn << 5) | rt);
 }
 
+static void
+gum_arm64_writer_put_load_store_pair_signed_offset (GumArm64Writer * self,
+                                                   GumArm64MemPairOperandSize op_size,
```

Comment:
Actually this line exceeds 80 cols, so that means the first argument should be moved to a separate line, 4 space indent, and the next arguments should be aligned to match it.